### ISSUE TITLE
String Index out of range

### ISSUE THREAD

**Issue by Towsty** [Morlok8k/MinecraftLandGenerator#10]
I'm not sure if this is still active or not. But figured I would post anyway

I get this error with every server I try this on. Let me know if you need more information.

Exception in thread "main" java.lang.StringIndexOutOfBoundsException: String index out of range: -77
        at java.lang.String.substring(String.java:1967)
        at morlok8k.MinecraftLandGenerator.Server.runMinecraft(Server.java:200)
        at morlok8k.MinecraftLandGenerator.Main.runCLI(Main.java:181)
        at morlok8k.MinecraftLandGenerator.Main.main(Main.java:132)

Basically, MLG worked perfectly on a heavily modded 1.10.2 server. About two weeks after that, I tried it again using the exact same settings on a new version of that pack. I've tried it on a couple of other packs since, not sure what the issue is but this always happens. 

Thoughts?

**Comment by Towsty** [comment]
So....after weeks I gave the whole setup a proverbial reboot. Clean install, works fine. :)
